src/uct/ib/rc/accel/rc_mlx5_ep.c

```diff
@@ -582,6 +582,11 @@ uct_rc_mlx5_ep_check(uct_ep_h tl_ep, unsigned flags, uct_completion_t *comp)
     UCT_CHECK_PARAM(comp == NULL, "Unsupported completion on ep_check");
     UCT_CHECK_PARAM(flags == 0, "Unsupported flags: %x", flags);
 
+    if (ucs_unlikely(!(ep->super.flags & UCT_RC_EP_FLAG_CONNECTED))) {
+        /* EP is not connected, just ignore keepalive request */
+        return UCS_OK;
+    }
```

Comment:
as discussed, it is not needed, will be fixed by ucp change

Reply:
assert fails due to progress makes shot into ep_check. added wireup stub to prevent ep_check till EP is connected